pkgs/native_assets_cli/CHANGELOG.md

```diff
@@ -17,8 +17,7 @@
 - No longer try to resolve uris encoded in `config.json` against any base uri.
   The `hook/{build,link}.dart` invoker has to ensure the uris it encodes can be
   opened as-is (i.e. without resolving against any base uri)
-- **Breaking change** Use unified classes instead of two `{OS,...}` and
-  move methods (e.g. from `OS`) to extensions (e.g. `OSLibraryNaming`)
+- **Breaking change** Use unified classes instead of an Api and a Model class.
```

Comment:
I had been interpreting the changelog mainly as for hook writers, not embedder writers.

From an API point of view, the exported classes mostly got extra methods (toJson, fromJson) and some methods moved out to extension methods. Maybe rephrase as:

* **Breaking change** Moved some methods to extension methods.
* Some classes in the `BuildConfig` and `BuildOutput` now expose `fromJson` and `toJson` in anticipation of ...

Reply:
Done.